[WEBSITE-606] Page titles

jakemulley opened the pull request.

Commit:
feature(page-titles): assign each view a page title

Commit:
fix(page-titles): revert Gemfile

Commit:
fix(page-titles): revert gemfile

mattrayner reviewed (changes requested):
If we're using title variables like this, perhaps the H1s should just have the title echoed out to remove duplication?

mattrayner commented on app/views/constituencies/current.html.haml:
```diff
@@ -1,3 +1,5 @@
+- title t('.title', current: t('current'), constituencies: t('constituencies_title'))
```
if we did something like - `page_title = title( t('.title', current: t('current'), constituencies: t('constituencies_title')) )`

Then you could do `%h1= page_title` to remove duplication

jakemulley replied:
Thanks @mattrayner - will do.

Commit:
feature(page-titles): remove duplication

mattrayner commented on Gemfile.lock:
```diff
@@ -1,6 +1,6 @@
 GIT
```
Can you reset your Gemfile.lock and run `bundle update pugin`

jakemulley replied:
This is the updated version.

Commit:
fix(page-titles): use display_name on page titles

Commit:
fix(page-titles): fix failing test for parties

Commit:
feature(page-titles): match page titles with H1s

fix(page-titles): revert Gemfile

fix(page-titles): use display_name on page titles

fix(page-titles): fix failing test for parties

Commit:
feature(page-titles): match page titles with H1s

Commit:
Merge branch 'feature/page-titles' of https://github.com/jakemulley/members-prototype into feature/page-titles

Commit:
Merge branch 'feature/page-titles' of https://github.com/jakemulley/members-prototype into feature/page-titles
